instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -28,13 +28,13 @@
     _start_internal_or_server_span,
     extract_attributes_from_object,
 )
+from opentelemetry.instrumentation.wsgi import add_response_attributes
 from opentelemetry.instrumentation.wsgi import (
-    add_custom_request_headers as wsgi_add_custom_request_headers,
+    collect_custom_request_headers_attributes as wsgi_collect_custom_request_headers_attributes,
 )
 from opentelemetry.instrumentation.wsgi import (
-    add_custom_response_headers as wsgi_add_custom_response_headers,
+    collect_custom_response_headers_attributes as wsgi_collect_custom_response_headers_attributes,
```

Comment:
You can combine both wsgi import statement in one import

Reply:
The imports are separated automatically by running `tox -e generate` (`isort` sorts/formats these imports). So, combining them will lead to build failure.